packages/chakra-components/src/MultiSelect/index.js

```diff
@@ -0,0 +1,743 @@
+import React, { forwardRef, createContext, useEffect, useRef } from "react";
+import {
+  Box,
+  PseudoBox,
+  Icon,
+  Spinner,
+  Tag,
+  TagLabel,
+  TagCloseButton,
+  useTheme,
+  useColorMode,
+} from "@chakra-ui/core";
+import { useForkRef } from "@chakra-ui/core/dist/utils";
+import Popper from "@chakra-ui/core/dist/Popper";
+import { useMultiSelectContext } from "./useMultiSelectContext";
+import { FixedSizeList as List } from "react-window";
+import AutosizeInput from "react-input-autosize";
+import { useComboBox } from "./useComboBox";
+import debounce from "lodash.debounce";
+import { useConstant } from "./useConstant";
+import { useAsyncFetching } from "./useAsyncFetching";
+
+import {
+  useMultiSelectStyle,
+  useMultiSelectListStyle,
+  useMultiSelectOptionStyle,
+  useMultiSelectInputStyle,
+} from "./styles";
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+export const MultiSelectContext = createContext();
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+const MultiSelect = forwardRef(
+  (
+    {
+      options,
+      defaultOptions,
+      cacheOptions,
+      value,
+      onChange,
+      filteredBy,
+      isListBox,
+      isMulti,
+      isAsync,
+      loadOptions = null,
```

Comment:
Shouldn't this be a function as default value ?

Reply:
Fixed this in the PR commit - https://github.com/timelessco/chakra-components/pull/78/commits/a1fdb01eb25215bde158911c64cdfabd94aabac8